apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/Message.tsx

```diff
@@ -0,0 +1,124 @@
+import dayjs from 'dayjs'
+import { kebabCase, noop, take } from 'lodash'
+import { Copy, FileDiff } from 'lucide-react'
+import ReactMarkdown from 'react-markdown'
+import { format } from 'sql-formatter'
+import { Badge, Button } from 'ui'
+import * as Tooltip from '@radix-ui/react-tooltip'
+
+import CodeEditor from 'components/ui/CodeEditor'
+
+const Message = ({
+  icon,
+  postedBy,
+  postedAt,
+  message,
+  onDiff = noop,
+  isDebug = false,
+}: {
+  icon: React.ReactNode
+  postedBy: string
+  postedAt?: number
+  message: string
+  isDebug?: boolean
+  onDiff?: (s: string) => void
+}) => {
+  return (
+    <div className="flex flex-col py-4 gap-4 border-t px-5">
+      <div className="flex flex-row gap-3 items-center">
+        {icon}
+
+        <span className="text-sm">{postedBy}</span>
+        {postedAt && (
+          <span className="text-xs text-foreground-muted">{dayjs(postedAt * 1000).fromNow()}</span>
+        )}
+        {isDebug && <Badge color="amber">Debug request</Badge>}
+      </div>
+      <ReactMarkdown
+        className="gap-2.5 flex flex-col"
+        components={{
+          p: ({ children }) => <div className="text-foreground-light text-sm">{children}</div>,
+          // intentionally rendering as pre. The other approach would be to render as code element,
+          // but that will render <code> elements which appear in the explanations as Monaco editors.
```

Comment:
total side note, can we turn of line highlighting like this in Monaco? 
Total distraction (occurring everywhere in Monaco, not just here)
https://share.cleanshot.com/CTFrkJD3

Reply:
are you referring to the borders around the line that the user is on? hmm might be up for a debate this one cause i think i prefer it as it lets me know which line my cursor is on really quickly